assets/javascripts/easter-egg.js

```diff
@@ -0,0 +1,18 @@
+const element = document.getElementById("heroMainImage");
+let iterations = 0;
+let timeout = null;
+
+element.addEventListener("click", event);
+
+function event() {
```

Comment:
```suggestion
function toggleEasterEgg() {
```

More descritptive name for the method